pyOCD/target/cortex_m.py

```diff
@@ -488,7 +488,10 @@ def writeMemory(self, addr, value, transfer_size = 32):
         write a memory location.
         By default the transfer size is a word
         """
-        self.transport.writeMem(addr, value, transfer_size)
+        try:
+            self.transport.writeMem(addr, value, transfer_size)
+        except:
```

Comment:
It is actually the ValueError exception that is being thrown. The HDK responds that it did not complete any transactions (transaction count = '0'). Perhaps the error checking order in dapTransfer should be changed so that TransferError is thrown first. I will look into this today.